include/libpldm/rde.h

```diff
@@ -23,13 +24,16 @@ extern "C" {
 #define PLDM_RDE_SCHEMA_URI_RESP_MAX_VAR_BYTES		    500
 #define PLDM_RDE_GET_RESOURCE_ETAG_REQ_BYTES		    4
 #define PLDM_RDE_GET_RESOURCE_ETAG_RESP_FIXED_BYTES	    4 // Include NULL
+#define PLDM_RDE_MULTIPART_SEND_REQ_FIXED_BYTES		    15
+#define PLDM_RDE_MULTIPART_SEND_RESP_BYTES		    2
 
```

Comment:
Need to define rdeOpID data structure , refer 5.3.25 section .,, whch is uint16 type and use this in the code.